### PR TITLE
try to improve release docs

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Specifically, try to slightly reorganize it in terms of what people are trying to achieve, rather than what type of release they ultimatly need to create.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: release/RELEASE.md [hunk 1]
```diff
@@ -1,124 +1,142 @@
...
-we'll assume you're patching the 1.18 branch to produce a snapshot (likely a
-release candidate) for 1.18.3.
 
+0. Make sure all the changes you want are in the release branch.
+1. Start _from latest `main`_ and run
+   ```
+   $ ./release.sh snapshot origin/release/2.0.x 2.0.1
+   ```
+   The output will be a line that starts with a commit sha, followed by a
+   snapshot release number. You need to take that line and add it to the [ LATEST`]
```
[the remark this answers is not in the document]
Comment (garyverhaegen-da) [reply release/RELEASE.md:57]: Thanks! Corrected.

File: release/RELEASE.md [hunk 2]
```diff
@@ -1,124 +1,142 @@
...
-1. The PR **must** be approved by a team lead before merging. As
-   of this writing (2022-02-08), @bame-da, @gerolf-da, @cocreature,
-   @ray-roestenburg-da or @adriaanm-da.
+In a perfect world, there is no need for a separate RC: the latest weekly
+snapshot can work. In the world we live in, though, we frequently want to add a
+few things, or patch previous minor rleases.
 
-1. Merge the PR and wait for the corresponding `main` build to finish.
+In those cases, we create `release/` branches (e.g. `release/2.0.x`). Those are
+special branches, protected by GiTHub rules and treated specially by CI.
```
Comment (tudor-da) [inline release/RELEASE.md:42]:
```suggestion
special branches, protected by GitHub rules and treated specially by CI.
```

File: release/RELEASE.md [hunk 3]
```diff
@@ -1,124 +1,142 @@
...
+   no other file. Add the `Standard-Change` label to that PR.
+3. When the PR is merged, the build of the corresponding commit on `main` will
+   create a "split release" bundle and push it to Artifactory. It should notify
+   on `#ci-failures-daml` on Slack.
+4. The next step is to request a build from the [Canton] team (on
+   `#team-canton`) that relies on the RC snapshot. Once you have a [canton]
+   release, you can proceed.
+5. Go to the [assembly] repo, and follow the instructions there to make a release
+   using the [Canton] version that was just created. The `LATEST` file on the
+   [assembly] repo only contains on version; it is safe to overwrite it, and to
```
Comment (tudor-da) [inline release/RELEASE.md:71]:
```suggestion
   [assembly] repo only contains one version; it is safe to overwrite it, and to
```

File: release/RELEASE.md [hunk 4]
```diff
@@ -1,124 +1,142 @@
...
-   PR with just the changed [ `LATEST`] file _to the `main` branch_. Add the
-   `Standard-Change` label to that PR.
-
+   The output will be a line that starts with a commit sha, followed by a
+   snapshot release number. You need to take that line and add it to the [`LATEST`]
+   file. You should put that line in the file so as to preserve semver
+   ordering, and overwrite any existing snapshot with the same prefix.
+2. Make a PR against the `main` branch with just that one line added, touching
+   no other file. Add the `Standard-Change` label to that PR.
 1. Once the PR has built, check that it was considered a release build by our
```
Comment (tudor-da) [inline release/RELEASE.md:108]:
```suggestion
2. Once the PR has built, check that it was considered a release build by our
```
and so forth?

File: release/RELEASE.md [hunk 5]
```diff
@@ -1,124 +1,142 @@
...
-
-1. The PR **must** be approved by a team lead before merging. As
-   of this writing (2022-02-08), @bame-da, @gerolf-da, @cocreature,
-   @ray-roestenburg-da or @adriaanm-da.
-
-1. Merge the PR and wait for the corresponding `main` build to finish. You
-   will be notified on `#team-daml`.
-
-1. Go to the [Testing](#testing) section of this file.
+3. When the PR is merged, the buil of the `main` branch will create the
```
Comment (tudor-da) [inline release/RELEASE.md:110]:
```suggestion
3. When the PR is merged, the build of the `main` branch will create the
```

File: release/RELEASE.md [hunk 6]
```diff
@@ -1,124 +1,142 @@
...
+- You should not be choosing an arbitrary commit, you should pick the latest RC
+  for the branch.
+- Go through the [checklist] before making the release. Note that
+  the checklist is not available publicly.
+- Instead of adding a line to [`LATEST`], remove the `-snapshot` part of the
+  version number for the existing RC.
+- Similarly, modifying the `LATEST` file on the assembly repo _should_ amount
+  to just removing the `-` parts of the version numbers for both Canton and
+  daml, but follow the instructions there.
+- You need a team lead to approve the PR on both the [daml] and [assembly] repos.
```
Comment (tudor-da) [inline release/RELEASE.md:93]: Do we need to change the GitHub release from pre-release to release for stable?

**Review by tudor-da — APPROVED** [review]
Great improvements. Some nitpicking and one open question

File: release/RELEASE.md (hunk 6 above)
Comment (garyverhaegen-da) [reply release/RELEASE.md:93]: Yes. I thought I'd put that in. Either I accidentally deleted that in a later edit round, or I'm losing my mind.

I'll address it.

File: release/RELEASE.md (hunk 4 above)
Comment (garyverhaegen-da) [reply release/RELEASE.md:108]: Markdown doesn't care, it corrects when rendering. But fair point, I'll make a pass through.

File: release/RELEASE.md [hunk 7]
```diff
@@ -1,124 +1,150 @@
...
+
+Once you have finished testing the release, coordinate with Product to decide
+how to communicate around it and when to remove the `prerelease` marker on the
+[releases page]. (This is what makes it available to `daml install`.)
```
Comment (garyverhaegen-da) [inline release/RELEASE.md:97]: @tudor-da Does this work?

Comment (tudor-da) [reply release/RELEASE.md:97]: 👍

**Pull request merged**